minos/scripts/install_cpc2.py

```diff
@@ -30,27 +30,28 @@ def main():
 	args.install_path = os.path.abspath(args.install_path)
 	
 	with cd(args.install_path):
-		cmd = "wget http://cpc2.cbi.pku.edu.cn/data/CPC2-beta.tar.gz"
+		cmd = "wget --no-check-certificate --content-disposition https://github.com/biocoder/CPC2/tarball/9a2b596"
```

Comment:
maybe add the hash as a command line arg to the install script. updates are not likely, but i could imagine submitting the py3 patch as a PR to the biocoda repo.

Reply:
Done